platform/debian/koreader.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+export LC_ALL="en_US.UTF-8"
+
+# open file, folder or home
+if [ -z "${1}" ]; then
+    ARGS="${HOME}"
+else
+    if [ -e "$(pwd)/${1}" ]; then
+        ARGS="$(pwd)/${1}"
+    elif [ -e "${1}" ]; then
+        ARGS="${1}"
+    fi
+fi
+
+# working directory of koreader
+KOREADER_DIR="/usr/local/share/koreader"
+
+# we're always starting from our working directory
+cd "${KOREADER_DIR}" || exit
+
+# export load library path for some old firmware
```

Comment:
This is only for some old Pocketbooks really; I probably more or less accidentally added it to the AppImage since I partially based it on the Ubuntu Touch port (while I suppose you based this on the AppImage).

See https://github.com/koreader/koreader/commit/327624e4fa1dd6bab765142995095905cd54c41a

Reply:
ok, on my machine it works if I remove `export LD_LIBRARY_PATH` but I see that everywhere. https://github.com/koreader/koreader/search?q=export+LD_LIBRARY_PATH&unscoped_q=export+LD_LIBRARY_PATH

Reply:
Good point, it might otherwise fail to resolve the correct libraries (i.e., ours). Looks like it was added as a precaution in https://github.com/koreader/koreader/commit/bb164c1deb87a7e0ac411e52eef293b915c6e978